.eslintrc

```diff
@@ -51,15 +51,7 @@
     "import/no-extraneous-dependencies": [
       "error",
       {
-        "devDependencies": [
-          "demo/**",
-          "test/**",
-          "test-jest/**",
-          "*.config.js",
-          "scripts/**",
-          "**/Spec.js",
-          "storybook/**"
-        ]
+        "devDependencies": true
```

Comment:
outside of PR scope, but hopefully a welcome change. I think ESLint may have gotten updated but we never updated the config

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md

Docs point out the new options.  Can no longer disable by folder